cl/simple_pages/templates/includes/available_fields.json

```diff
@@ -350,5 +350,61 @@
       "parentheticals":false,
       "recap_archive":true,
       "oral_arguments":false
+   },
+  {
+      "field":"jurisdictionType",
+      "description":"Stands for jurisdiction in RECAP XML docket. For example, 'Diversity', 'U.S. Government Defendant'.",
+      "opinions":false,
+      "parentheticals":false,
+      "recap_archive":true,
+      "oral_arguments":false
+   },
+  {
+      "field":"chapter",
+      "description":"In bankruptcy dockets, the chapter the bankruptcy is currently filed under.",
+      "opinions":false,
+      "parentheticals":false,
+      "recap_archive":true,
+      "oral_arguments":false
+   },
+  {
+      "field":"trustee_str",
+      "description":"In bankruptcy dockets, the name of the trustee handling the case.",
+      "opinions":false,
+      "parentheticals":false,
+      "recap_archive":true,
+      "oral_arguments":false
+   },
+  {
+      "field":"entry_number",
+      "description":"# on the PACER docket page. For appellate cases, this may be the internal PACER ID for the document, when an entry ID is otherwise unavailable.",
```

Comment:
```suggestion
      "description":"The entry number on the PACER docket page. For appellate cases, this may be the internal PACER ID for the document, when an entry ID is otherwise unavailable.",
```